linodecli/baked/operation.py

```diff
@@ -338,20 +363,29 @@ def parse_args(
             for arg in self.args:
                 if arg.read_only:
                     continue
+
+                arg_type = (
+                    arg.item_type if arg.datatype == "array" else arg.datatype
+                )
+                arg_type_handler = TYPES[arg_type]
+
+                if arg.nullable:
+                    arg_type_handler = wrap_parse_nullable_value(arg_type)
+
```

Comment:
I feel it would be nicer if we can wrap all these logic into an function. But this is just my opinion so it's an optional suggestion!

Reply:
Good idea!